start-up/moderation-tutorial.md

```diff
@@ -1,3 +1,69 @@
 # Moderation Tutorial
 
-TODO
\ No newline at end of file
+## Intro
+
+This guide will give you the basic information needed to moderate your Discord server with MoonlightBot. It includes a list of commands and how they work, complete with their arguments, infractions, and log information to help you better understand how to use them
+
+## Arguments
+
+Arguments are options you may use while executing moderation commands. You can find a more in-depth guide about arguments in the [arguments page](/start-up/arguments.md)
+
+![](/.gitbook/assets/MuteArguments.png)
+
+- **`user`**: The target member of the command
+- **`reason`**: The reason for which you are executing the command
+- **`notify`**: Whether or not the bot will message the user after the command is executed
+- **`no-infraction`**: Whether or not the command will count towards the user's infractions
+
+{% hint style="info" %}
+If both the `notify` and `no-infraction` arguments are true, the target will not be notified in order to prevent harrassment via MoonlightBot
+{% endhint %}
+
+- **`duration`**: The amount of time for which the user will be affected by the command. For more information about this argument, refer to the [arguments page](/start-up/arguments.md#durations)
+- **`remove-roles`**: Exclusive to the mute commands, this argument will remove all of the user's roles except the muterole if true
+- **`role`**: The role to assign/remove in a role command
+
+## Moderation Tools
+
+These features help ensure a seamless moderation experience with MoonlightBot
+
+### Infractions
+
+![](/.gitbook/assets/InfractionsExample.png)
+
+Infractions help staff members organize actions taken while moderating, such as mutes. This is a versatile tool with multiple submcommands to assist your moderation needs. More information about infractions can be found in the [infractions page](/moderation-commands/infractions.md)
```

Comment:
I'd reword, more important to state they're to help moderators keep track of a user's bad conduct and make informed decisions when moderating the server